common/src/debug_common.C

```diff
@@ -38,16 +38,16 @@ int common_debug_dwarf = 0;
 int common_debug_addrtranslate = 0;
 int common_debug_lineinfo = 0;
 int common_debug_parsing = 0;
+int common_debug_initialized = 0;
```

Comment:
This is no longer threadsafe. Is that a problem for parallel DWARF parsing?

Reply:
I agree with your assessment, but It was not completely data race free to begin with. The existing debug variables are also global variables. I don't intend to make large changes. 

If I have to guess, I think maybe because we will only set these debug variables from 0 to 1, and never change them from 1 to other values. Such data races are benign: more than one threads may perform initialization, which will check environment variable and set `common_debug_initialized`. Given the initialization is idempotent, we are fine here.

Reply:
Ah, ok. I wasn't sure what all was going on in the init function. This change is fine, then.